internal/web_server/web/views/components/dashboard_components/access_settings_components/access_settings_components.templ

```diff
@@ -0,0 +1,144 @@
+package access_settings_components
+
+import (
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/icons"
+	"github.com/Molnes/Nyhetsjeger/internal/models/users/access_control"
+	"fmt"
+)
+
+const (
+	_ADD_ADMIN_FORM    = "add-admin-form"
+	_ADMINS_TABLE      = "admins-table"
+	_NO_ADMINS_MESSAGE = "no-admins-message"
+	_ADMINS_TABLE_BODY = "admin-table-body"
```

Comment:
`_ADMINS_TABLE_BODY` text does not match `admin-table-body`. Intentional? `admins` -> `admin`

Reply:
2e6be74a6cb112831d964dd0d5b2c575a9ffdaea